ddtrace/contrib/django/compat.py

```diff
@@ -2,10 +2,26 @@
 
 
 if django.VERSION >= (1, 10, 1):
+    from django.urls import get_resolver
+
     def user_is_authenticated(user):
         # Explicit comparision due to the following bug
```

Comment:
```suggestion
        # Explicit comparison due to the following bug
```

Reply:
That is ok, with how I spell/type it is going to come in handy when reviewing my PRs